bridges/ReutersBridge.php

```diff
@@ -1,101 +1,198 @@
+            $data = $content['content'];
+            // This will check whether that content is a image URL or not.
+            if (
+                strpos($data, '.png') !== false ||
+                strpos($data, '.jpg') !== false
+            ) {
+                $description = $description . "<img src=\"$data\">";
+            } else {
+                if ($content['type'] == 'inline_items') {
+                    //Fix issue with some content included brand name or company name.
+                    $item_list = $content['items'];
+                    $description = $description . '<p>';
+                    foreach ($item_list as $item) {
+                        $description = $description . $item['content'];
+                    }
+                    $description = $description . '</p>';
+                } else {
+                    if (
+                        strtoupper($data) == $data ||
+                        $content['type'] == 'heading'
+                    ) {
+                        //Add heading for any part of content served as header.
+                        $description = $description . "<h3>$data</h3>";
+                    } else {
+                        $description = $description . "<p>$data</p>";
+                    }
+                }
+            }
+        }
+
+        $content_detail = [
+            'content' => $description,
+            'author' => $author,
+        ];
+        return $content_detail;
+    }
+
+    public function collectData()
+    {
+        $feed = $this->getInput('feed');
+        $data = $this->getJson($feed);
+        $reuters_wireitems = $data['wireitems'];
+        $this->feedName = $data['wire_name'] . ' | Reuters';
+        $processedData = $this->processData($reuters_wireitems);
+
+        // Merge all articles from Editor's Highlight section into existing array of templates.
+        $top_section = reset($reuters_wireitems);
+        if ($top_section['wireitem_type'] == 'headlines') {
+            $top_articles = $top_section['templates'][1]['headlines'];
+            $processedData = array_merge($top_articles, $processedData);
+        }
+
+        foreach ($processedData as $story) {
+            $item['uid'] = $story['story']['usn'];
+            $article_uri = $story['template_action']['api_path'];
+            $content_detail = $this->getArticle($article_uri);
+            $description = $content_detail['content'];
+            $author = $content_detail['author'];
+            $item['author'] = $author;
+            if (!(bool) $description) {
+                $description = $story['story']['lede']; // Just in case the content doesn't have anything.
+            }
+            #	$description = $story['story']['lede'];
+            $image_url = $story['image']['url'];
+            if (!(bool) $image_url) {
+                $image_url =
+                    'https://s4.reutersmedia.net/resources_v2/images/rcom-default.png'; //Just in case if there aren't any pictures.
```

Comment:
Is adding the default Reuters logo image to image-less news stories better than just not having images in those stories?

Reply:
Yep, it's better than not to have that.

Reply:
Fixed that in the latest commit.